src/main/java/org/web3j/protocol/core/personal/Personal.java

```diff
@@ -0,0 +1,33 @@
+package org.web3j.protocol.core.personal;
+
+import java.math.BigInteger;
+
+import org.web3j.protocol.Web3jService;
+import org.web3j.protocol.core.Request;
+import org.web3j.protocol.core.personal.methods.response.EcRecovered;
+import org.web3j.protocol.core.personal.methods.response.ImportedRawKey;
+import org.web3j.protocol.core.personal.methods.response.LockedAccount;
+import org.web3j.protocol.core.personal.methods.response.NewAccount;
+import org.web3j.protocol.core.personal.methods.response.Signed;
+import org.web3j.protocol.core.personal.methods.response.Transaction;
+import org.web3j.protocol.core.personal.methods.response.UnlockedAccount;
+import org.web3j.protocol.parity.methods.response.PersonalListAccounts;
+
+public interface Personal{
```

Comment:
> Those calls that are common to both Parity & Geth can be in org.web3j.protocol.admin.Personal

do you mean that parity is to be moved into that package as well? wouldn't it be better that  they all were located in org.web3j.protocol.geth.Geth?